### ISSUE TITLE
Health of API and debug API should be websocket connection

### ISSUE THREAD

**Issue by vojtechsimetka** [ethersphere/bee-status#17]
Both standard API and debug API should have health endpoint which would be websocket. This way the bee-status can monitor the endpoint and alert user if anything goes wrong.

**Comment by agazso** [comment]
These would require changes from the Bee side too. We can have this functionality by setting up a timer that checks the health endpoints without websocket.

**Comment by vojtechsimetka** [comment]
Yes that is the ultimate idea - to have a discussion about such endpoint on Bee.

Great suggestion with the polling solution.

**Comment by acud** [comment]
On bee side - not sure, since k8s is polling these endpoints too, it means that they might need to be implemented to support both ws and normal json responses. Which is probably doable. Not sure how urgent this is though.

**Comment by Eknir** [comment]
@vojtechsimetka , if this functionality can be implemented also on the js-side, I would suggest to do it like that for now.
